apps/network-alerts/js/index.js

```diff
@@ -31,23 +47,25 @@
       return;
     }
 
-    var req = navigator.mozSettings.createLock().get(CMAS_ENABLED_KEY);
+    var sendAlertPromise =
+      isPresidentialAlert(message) ?
+      Promise.resolve(true) :
+      isCmasEnabledForServiceId(message.serviceId).catch((e) => {
+        console.error('CMAS: Unable to query settings database', e);
+        return false;
+      });
```

Comment:
This structure looks good, just a small nit that `sendAlertPromise` looks like we actually send the alert and return the sent status. Maybe `checkAlertSendablePromise` or something else would be more readable?